geograypher/meshes/meshes.py

```diff
@@ -15,8 +15,9 @@
 import rasterio as rio
 import shapely
 import skimage
+import ubelt as ub
 from shapely import MultiPolygon, Polygon
-from skimage.transform import resize
+from skimage.transform import resize  # might look into for cache
```

Comment:
I avoid comments that aren't relevant to that section of code. If you think it's important to remember, maybe add it as a TODO near the caching?

Reply:
Sorry forgot to remove this from before